.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: php
+
+php:
+  - 5.3.3
```

Comment:
FYI, I don't mind bumping to 5.4 directly, but I don't have full control here.

This should indeed be 5.3, not 5.3.3